tests/garage/torch/algos/test_dqn.py

```diff
@@ -0,0 +1,109 @@
+"""Test DQN performance on cartpole."""
```

Comment:
can you add a test which ensures that you can resume the algorithm from a snapshot?

Reply:
done.